docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,69 @@
+*******************************************************
+Storage Quota at /home/
+*******************************************************
+Staying under your ``/home/<username>`` quota is vital in preventing issues using Open OnDemand and performing tasks on Discovery. Here are some suggestions to stay under quota and general best practices with using your ``/home/<username>`` directory.
+
+Analyze disk usage
+=======================================================
+From a compute node, ``srun --pty /bin/bash``, run the following command from your ``/home/<username>`` directory: ::
+    
+ du -shc .[^.]* *
+
+This command will output the size of each file, directory, and hidden directory in your ``/home/<username>`` space, with the total of your ``/home`` directory being the last line of the output. After determining the large files and directories, you can move them to the appropriate location, (g.g., ``/work`` for research) or you can back up and delete the files and directories if they are no longer required.
```

Comment:
Would it make sense to change: "After **determining** the large files and directories" to "After **identifying** the large files and directories?"